packages/lib-classifier/dev/components/App/App.js

```diff
@@ -1,12 +1,36 @@
 import React from 'react'
+import { panoptes } from '@zooniverse/panoptes-js'
 
 import Classifier from '../../../src/components/Classifier'
-import project from './projectFixture'
 
 class App extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      project: null
+    }
+  }
+
+  async componentDidMount () {
+    const id = '335'
+    try {
+      const response = await panoptes.get(`/projects/${id}`)
```

Comment:
I messed up the module export on projects requests. I noticed today while working on a branch on panoptes.js. Gonna fix it in the tutorial resource branch 😬 . This works though 👍

Reply:
I wondered why it wasn't working, I thought it was me 😛